app/views/index.blade.php

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8">
+		<title>Laravel PHP Framework</title>
+		<style>
+			@import url(//fonts.googleapis.com/css?family=Lato:700);
+
+			body {
```

Comment:
as GreatDeveloper you should take care about everything. To make beautiful and efficient code